### ISSUE TITLE
Run coordinator as routers part

### ISSUE THREAD

**Issue by reshke** [pg-sharding/spqr#279]
In the begging of SPQR project, the decision to have coordinator as separate unit was made. 

This was not a right, but neither a wrong decision. On one hand, this helps to manage installation topology and keep routers metadata in sync, avoid data loss and routing schema corruption. 

On other hand, why should we have this unit as separate binary being run by admin, robot, etc? 

Maybe run this as part of router process will be better and will simplify SPQR a lot for users. 

TLDR:
simply run  this [function](https://github.com/pg-sharding/spqr/blob/master/cmd/coordinator/main.go#L22) from [here](https://github.com/pg-sharding/spqr/blob/master/cmd/router/main.go#L169) in separate goroutine

**Comment by Denchick** [comment]
I like the idea. We discussed with @x4m, @munakoiso, and @EinKrebs how we should roll out the coordinator. We concluded that it would be easier to combine the coordinator and the router. We haven't decided where the QDB will be launched yet, but by combining these pieces, we can reap the benefits.

Anyway, there are some disadvantages in this desicion, right?
The only thing that confuses me is that @diphantxm has already invested time into fixing bugs and writing tests specifically for the coordinator. If we merge the coordinator into the router now, it might seem like some of that work has gone to waste.

**Comment by reshke** [comment]
>it might seem like some of that work has gone to waste.

Dont think any work is going to waste at all.
